docs/pages/versions/unversioned/sdk/app-integrity.mdx

```diff
@@ -0,0 +1,186 @@
+---
+title: App Integrity
+description: A library that provides access to Apple's App Attest service on iOS and Google's Play Integrity API on Android.
+sourceCodeUrl: 'https://github.com/expo/expo/tree/main/packages/expo-app-integrity'
+packageName: '@expo/app-integrity'
+platforms: ['ios', 'android']
+iconUrl: '/static/images/packages/expo-maps.png'
+isAlpha: true
+---
+
+import APISection from '~/components/plugins/APISection';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Terminal } from '~/ui/components/Snippet';
+import { CODE } from '~/ui/components/Text';
+import { ConfigReactNative } from '~/ui/components/ConfigSection';
+
+`@expo/app-integrity` provides APIs to help ensure your backend resources are accessed only by legitimate installations of your app running on genuine devices. It uses Apple's [App Attest service](https://developer.apple.com/documentation/devicecheck/establishing-your-app-s-integrity) on iOS and Google's [Play Integrity APIs](https://developer.android.com/google/play/integrity) on Android to verify app authenticity, helping prevent unauthorized clients, modified apps, or automated scripts from making requests to your server.
+
+In simple terms, `@expo/app-integrity` helps your server tell the difference between:
+
+- Your **real app** running on a **real device**, and
+
+- Anything else (modified apps, scripts, emulators).
+
+It does this by using the platform-recommended app attestation services.
+
+## Installation
+
+<APIInstallSection />
+
+## Usage on iOS
```

Comment:
Let's move the entire Usage on iOS section after Android as per the platform order stated in our writing style guide.